### ISSUE TITLE
Doesn't show details of activities amazfitbip 

### ISSUE THREAD

**Issue by giupardeb** [Freeyourgadget/Gadgetbridge#1566]
#### Before reporting a bug, please confirm the following:
- [x] I have read the [wiki](https://github.com/Freeyourgadget/Gadgetbridge/wiki), and I didn't find a solution to my problem / an answer to my question.
- [x] I have searched the [issues](https://github.com/Freeyourgadget/Gadgetbridge/issues), and I didn't find a solution to my problem / an answer to my question.
- [x] If you upload an image or other content, please make sure you have read and understood the [github policies and terms of services](https://help.github.com/articles/github-terms-of-service/#1-responsibility-for-user-generated-content)

#### Your issue is:
Hi all, Today I synchronized an activities. When I tap an element of the activities list, I doesn't show details about activity(show attachment). While on device I saw details of activity. Why?

#### Your wearable device is: 
amazfit bip FW:1.1.5.56 ; GPS: 18344,eb2f43f,126 ; HW: V0.11.19.5

#### Your android version is:
 LineageOS 15.1(Android 8.1)

#### Your Gadgetbridge version is: 
0.34.1

![screen](https://user-images.githubusercontent.com/4471950/61856105-c0518580-aec1-11e9-9624-a4847b6f8ca6.jpg)

**Comment by vanous** [comment]
Ypu must open the activity (gpx file) with a compatible application, for example osmand or aat.

**Comment by ashimokawa** [comment]
You mean the summary, etc?
Because it is not implemented in Gadgetbridge due to the lack of developers time.

**Comment by giupardeb** [comment]
@ashimokawa yes! I want to see the summary..it's an important feature. we hope they implement it soon

**Comment by giupardeb** [comment]
> You must open the activity (gpx file) with a compatible application, for example osmand or aat. It works here and it's pretty cool.

what is aat application? However, with osmand I don't see kcal burned, and there is a mismatches from amazfit bip and osmand (time doesn't match, kilometers doens't match, etc..)

**Comment by petrvanekrobe** [comment]
Aat is Another Android Tracker, available via F-droid. It shows kcal, but I think it calculates it from the data. I can see small discrepancy in time and distance, but it's quite negligible. Look below, data from the watch and from Aat, osmAnd. If I understand correctly, the file is fetched as is. Perhaps the difference is in the time of starting activity and gps getting fix...? 

![20190802_095634_1](https://user-images.githubusercontent.com/45997318/62354728-d25ca500-b50d-11e9-9a71-3d1ae78cdaea.jpg)
![20190802_095616_1](https://user-images.githubusercontent.com/45997318/62354743-d983b300-b50d-11e9-8448-9e05501908d5.jpg)
![Screenshot_20190802-100529_AAT_1](https://user-images.githubusercontent.com/45997318/62354767-e43e4800-b50d-11e9-92cb-738b3191d1e8.jpg)
![Screenshot_20190802-095507_OsmAnd~_1](https://user-images.githubusercontent.com/45997318/62354811-f28c6400-b50d-11e9-9d1a-88958ffbcfe4.jpg)
![Screenshot_20190802-095511_OsmAnd~_1](https://user-images.githubusercontent.com/45997318/62354823-fb7d3580-b50d-11e9-9874-d7d0045dc244.jpg)

**Comment by giupardeb** [comment]
AAT is very cool and it would seem very precise! I insall it now

**Comment by vanous** [comment]
Aat is Another Android Tracker, available via F-droid. It shows kcal, but I think it calculates it from the data. I can see small discrepancy in time and distance, but it's quite negligible. Look below, data from the watch and from Aat, osmAnd. If I understand correctly, the file is fetched as is. Perhaps the difference is in the time of starting activity and gps getting fix...? 

![20190802_095634_1](https://user-images.githubusercontent.com/45997318/62354728-d25ca500-b50d-11e9-9a71-3d1ae78cdaea.jpg)
![20190802_095616_1](https://user-images.githubusercontent.com/45997318/62354743-d983b300-b50d-11e9-8448-9e05501908d5.jpg)
![Screenshot_20190802-100529_AAT_1](https://user-images.githubusercontent.com/45997318/62354767-e43e4800-b50d-11e9-92cb-738b3191d1e8.jpg)
![Screenshot_20190802-095507_OsmAnd~_1](https://user-images.githubusercontent.com/45997318/62354811-f28c6400-b50d-11e9-9d1a-88958ffbcfe4.jpg)
![Screenshot_20190802-095511_OsmAnd~_1](https://user-images.githubusercontent.com/45997318/62354823-fb7d3580-b50d-11e9-9874-d7d0045dc244.jpg)

**Comment by vanous** [comment]
@giupardeb Cool. I like it too.
